addons/portal/views/portal_templates.xml

```diff
@@ -75,6 +75,8 @@
 
     <template id="portal_layout" name="Portal Layout">
         <t t-call="portal.frontend_layout">
+            <t t-set="is_portal" t-value="True"/>
```

Comment:
Mmh, in fact it was only needed here... good :)

Reply:
Yep, quite a relief 😋